roles/tpa_single_node/templates/manifests/importer/DeploymentB.yaml.j2

```diff
@@ -1,135 +0,0 @@
-apiVersion: apps/v1
```

Comment:
Please restore and optimize the configuration in a separate PR, there are missing placeholder in the Deployment.yaml.j2

Reply:
Thanks @desmax74, I've fixed the Deployment template, it was effectively missing the variable for the manifest name.

I fully tested `single` and `multiple` cases